src/network/netdev/vxlan.c

```diff
@@ -416,6 +426,10 @@ static int netdev_vxlan_verify(NetDev *netdev, const char *filename) {
                                               "The local address cannot be '%s' when Independent= is enabled, ignoring.",
                                               strna(netdev_local_address_type_to_string(v->local_type)));
 
+        if (v->vnifilter && !v->external) {
+                log_netdev_warning(netdev, "VNIFilter= is enabled while External= is disabled. Will be ignored.");
```

Comment:
Could you explicitly mention what is ignored?
E.g. `"VNIFilter= setting will be ignored."`